time stamp
uint32 image_height
uint32 image_width
uwb_uvdar_fusion/Point2DWithFloat[] points
